content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -578,6 +579,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
   completions without relying on Pods remaining in the cluster indefinitely.
   The Job controller uses Pod finalizers and a field in the Job status to keep
   track of the finished Pods to count towards completion.
+- `KMSv2`: Enables KMS v2 API for encryption at rest.
```

Comment:
Can we link to the KMS docs?

Reply:
opened https://github.com/kubernetes/website/pull/38164 to add the link